debian/rules

```diff
@@ -43,15 +43,15 @@ binary-syncd-rpc:
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
+    ifeq ($(shell cat /tmp/syncd-build), --enable-rpcserver=yes)
```

Comment:
Yes. I tried that. Unfortunately, it appears my files get copied before the `binary` target is called, so my files get copied before the sed command executes. I moved the sed command to `override_dh_auto_configure` because it technically a configuration step.